firefox-ios/Client/Frontend/Browser/Toolbars/AddressBarAddTabView.swift

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+import Common
+
+class AddressBarAddTabView: UIView, ThemeApplicable {
+    private struct UX {
+        static let cornerRadius: CGFloat = 12.0
+    }
+
+    let plusIconView: UIImageView  = .build {
+        $0.alpha = 0.0
+        $0.image = UIImage(named: StandardImageIdentifiers.Large.plus)
+    }
+
+    init() {
+        super.init(frame: .zero)
+        setup()
```

Comment:
Usually we name this `setupLayout` or `setupView` - can we stick to this naming convention here?